README.md

```diff
@@ -51,15 +52,13 @@ SD card file size 2Gb zipped to 1.6Gb
 ## Instructions: (READ FULLY BEFORE FOLLOWING)
 
 1. Install necessary dependcies listed using guide above. Don't forget that optional deps will bring fast downloads.
-2. Use install script to obtain FPP
-3. Move Brawl into FasterProjectPlus/bin/Games directory
-4. Run the game by opening the .elf file in dolphin
+2. Use install script to obtain FPP.
+3. Move Brawl into FasterProjectPlus/bin/Games directory.
+4. Run the game by opening the .elf file in Ishiiruka.
 
 ## To use:
 
-```sh
-sh -c "$(curl -Ls https://github.com/Birdthulu/FPM-Installer/raw/master/setup)"
-```
+`curl -Ls https://github.com/Birdthulu/FPM-Installer/raw/master/setup && ./setup`
```

Comment:
In my testing this does not work if someone doesn't already have a file called setup on their computer

Reply:
Agreed, the most staightforward workaround is to use wget instead.